source/includes/usage-examples/code-snippets/findOne.go

```diff
@@ -33,11 +34,38 @@ func main() {
 		}
 	}()
 
+	type Address struct {
+		Building    string
+		Coordinates [2]float64 `bson:"coord"`
+		Street      string
+		Zipcode     string
+	}
+
+	type Grades struct {
+		Date  primitive.DateTime
+		Grade string
+		Score int
+	}
```

Comment:
You dont have to define these. See the below comment for how to wrap these in an `interface{}` for brevity.